### PR TITLE
Center the search bar, no matter how many panels are open

### PR DESCRIPTION
**Pull request by hamburge**
Changes the class of main based on how many panels are open. Left margin of search bar changes based on that class.

@designedbyscience Is this along the lines of what you were suggesting?

issue #1892

### CODE REVIEW

File: src/style/search/search-bar.less [hunk 1]
```diff
@@ -44,6 +43,19 @@
     overflow: visible;
 }
 
+.main__no-panel .search-bar__dialog {
+    margin-left: -@search-bar-width / 2;
+}
+
+
+.main__one-panel .search-bar__dialog {
+    margin-left: -(@properties-width / 2)  - (@search-bar-width / 2);
```
Comment (iwehrman) [inline src/style/search/search-bar.less]: Isn't `@properties-width` going to vary at runtime?

Comment (designedbyscience) [inline src/style/search/search-bar.less]: No, it is a CSS variable (it is the width of one panel...maybe we should rename that)

Comment (designedbyscience) [inline src/style/search/search-bar.less]: Also, extra line

**Comment by designedbyscience** [comment]
Yep, this is what I was thinking. Could you go rename `@properties-width` to `@panel-column-width`? One other nit about lines

**Comment by hamburge** [comment]
@designedbyscience Great! Made those changes.

**Comment by designedbyscience** [comment]
@hamburge In the 2 column case, I think you are 0.5rem off center (it calculates -50rem but it should be -50.5rem;

**Comment by hamburge** [comment]
@designedbyscience Got it, subtracted .5rem-- doesn't seem to be stored anywhere as a CSS variable.

**Comment by hamburge** [comment]
Not ready yet-- just realized that a rogue commit got in here...

**Comment by hamburge** [comment]
Ok removed the unrelated commit

**Comment by designedbyscience** [comment]
:guitar: on

**Pull request merged**